ivy/array/general.py

```diff
@@ -327,6 +327,42 @@ def to_list(self: ivy.Array):
         """
         return ivy.to_list(self)
 
+    def inplace_decrement(
+        self: Union[ivy.Array, ivy.NativeArray], val: Union[ivy.Array, ivy.NativeArray]
+    ) -> ivy.Array:
+        """ivy.Array instance method variant of ivy.inplace_decrement. This method simply
```

Comment:
Refer [lint check](https://github.com/unifyai/ivy/runs/7839381090?check_suite_focus=true#step:5:7) and fix it.

Reply:
Hi @ankitdsi2010, thanks for your comment. Lint tests are passing OK now. Best regards, Elena.